<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Platform</class>
 <widget class="QWidget" name="Platform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1021</width>
     <height>651</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>酒店列表</string>
    </attribute>
    <widget class="QTableWidget" name="table1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>1011</width>
       <height>451</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>221</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>待审核酒店</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelHotelNum">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnApproveHotel">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>30</y>
       <width>131</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>通过审核</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSaveAndExit">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>30</y>
       <width>141</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>保存退出</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRefreshtable1">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>30</y>
       <width>131</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>刷新列表</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSTOP">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>30</y>
       <width>121</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>停业整改</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>房间列表</string>
    </attribute>
    <widget class="QTableWidget" name="table2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>1001</width>
       <height>471</height>
      </rect>
     </property>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>100</number>
     </attribute>
    </widget>
    <widget class="QPushButton" name="btnApproveRoom">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>30</y>
       <width>181</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>通过审核</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>241</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>待审核房间</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelNum2">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRefreshtable2">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>30</y>
       <width>181</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>刷新列表</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>订单列表</string>
    </attribute>
    <widget class="QTableWidget" name="table3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>1011</width>
       <height>481</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>241</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="text">
      <string>待审核退款</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelRefundNum">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>50</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnApproveOrder">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>20</y>
       <width>171</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>通过审核</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRefreshtable3">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>20</y>
       <width>171</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>刷新列表</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>用户列表</string>
    </attribute>
    <widget class="QTableWidget" name="table4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>1011</width>
       <height>471</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>201</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="text">
      <string>用户列表</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDisable">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>30</y>
       <width>171</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>注销用户</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRefreshTable4">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>30</y>
       <width>151</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>刷新列表</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
